server/src/bots/telegram/utils/messageRegistry.ts

```diff
@@ -0,0 +1,14 @@
+export class MessageRegistry
+{
+    _bot: any;
```

Comment:
how about using unknown instead of any?

Reply:
_bot: unknown;

Reply:
intellisense tells it's not ok, since we are calling on method